### DISCUSSION TITLE
Relationships

### DISCUSSION THREAD

**Discussion by hugopvl** [directus/directus#8122] [Archive]
Hey guys,

A question about relationships, I have an item that we imagine is the recipe

_recipe collection:_
**(recipe01):**

Then I have several items that are spices,

_spice collection:_
**spice1** - (**recipe01**)
**spice2** - (**recipe01**)
**spice3** - (**recipe01**)

So I create an M2M relationship

In each item of spices I can see which recipe it was used in, it's perfect, 
but even better would be to show in each recipe the spices it uses, is this possible? :)

**Comment by rijkvanzanten** [comment]
> but even better would be to show in each recipe the spices it uses, is this possible? :)

You can setup another m2m field from recipe->spices using the same relationship setup 🙂

**Reply by hugopvl** [reply]
Sorry, or I'm doing something wrong, or it doesn't work :/

Another experiment I'm doing,
 
-a style collection,
-a prints collection,
 
a style can have several prints, that is, each time I create a print, I have to indicate which style it belongs to.

![image](https://user-images.githubusercontent.com/81326296/133982517-48a6b629-b4c0-450e-b1cf-7116e6a4b33b.png)

![image](https://user-images.githubusercontent.com/81326296/133984905-a9f1acfd-5b70-48b8-bde4-b7764582eb57.png)

And it works perfectly.

Now what I wanted and I don't know if it's possible, is that the sytle shows up all the corresponding prints automatically.

If I create another relationship, m2m style_prints creates another collection and I'll have to repeat information, select all prints corresponding to that style.

![image](https://user-images.githubusercontent.com/81326296/133986484-4adbf9f9-b119-434a-94b7-5a74c0c01b25.png)
![image](https://user-images.githubusercontent.com/81326296/133986700-ad7e0c5c-809d-4300-9242-84f873f68ee8.png)

![image](https://user-images.githubusercontent.com/81326296/133986550-359e499f-ff43-49c4-89c7-99a7d3319c8a.png)

Thanks

**Reply by rijkvanzanten** [reply]
@hugopvl you should be able to uncheck "Auto-fill" when setting up the relationship, and then re-use the existing "prints_styles" collection

**Reply by hugopvl** [reply]
I had already tried to do this, but the strange thing is that when I do this, the item (print) gives an error (page not found). and the option relationship desapear and the interface looks like this:

![image](https://user-images.githubusercontent.com/81326296/134027984-bcb1dcec-f88a-4e52-8119-b55d2c963ea3.png)


in style apear the print, but when you click give an error:

![image](https://user-images.githubusercontent.com/81326296/134028434-818b2dbf-ed6d-424d-85e6-906ded41d82c.png)

![image](https://user-images.githubusercontent.com/81326296/134028470-ba1aae9f-3eba-4476-9906-f7327979e279.png)

the config stay ok 

![image](https://user-images.githubusercontent.com/81326296/134028558-41b15edb-14fe-46ff-9fdc-21c785906369.png)

**Comment by hugopvl** [comment]
Im sorry, but now its working... with this option: 

working perfectly 

sorry but i was a bit blind :P

![image](https://user-images.githubusercontent.com/81326296/134029607-138ddc70-8d51-4d87-a634-d1e4cf8c4c96.png)
